django/templatetags/static.py

```diff
@@ -98,6 +98,13 @@ def __init__(self, varname=None, path=None):
         self.path = path
         self.varname = varname
 
+    def __repr__(self) -> str:
+        return "<%s path='%s' varname='%s' >" % (
```

Comment:
If you look at `__init__`, this is a simple class that is easy to reconstruct, so we don't need to use the `<…>` pattern. Also consider that `varname` could be `None` so shouldn't be manually quoted like this, get it's repr instead. I'd change this to the following:

```python
        return f'{self.__class__.__name__}(path={self.path!r}, varname={self.varname!r})'
```

Reply:
thank you for reviewing,  I'm not sure that I get the last point about `varnam` . do you mean? I should use `repr() `function on `varnam`